assets/styles/index.scss

```diff
@@ -1,5 +1,5 @@
 $icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
-@import '~bootstrap-sass/assets/stylesheets/bootstrap';
+@import '~bootstrap-sass/assets/stylesheets/bootstrap/';
```

Comment:
I was playing around with the import because i couldn't get it to work. the / was residual result of that. didn't mean to be there.